tests/query_test/qpattern_test.cpp

```diff
@@ -0,0 +1,2298 @@
+//
+// Created by kaimal on 14.11.21.
+//
+#include <doctest/doctest.h>
+#include <rdf4cpp/rdf.hpp>
+
+using namespace rdf4cpp::rdf;
+
+static void test(Node graph, Node sub, Node pred, Node obj, bool result){
+
+    auto qpattern = query::QuadPattern {graph, sub, pred, obj};
+
+    SUBCASE("Check validity & output"){
+        CHECK(qpattern.graph() == graph);
+        CHECK(qpattern.subject() == sub);
+        CHECK(qpattern.predicate() == pred);
+        CHECK(qpattern.object() == obj);
+
+        CHECK(qpattern.valid());
```

Comment:
This line doesn't make sense but the following two lines do. Remove?

Reply:
Changed everything as per request except pt no.4 (one add_executable per tests_{classname}.cpp. Executable name: tests_{classname} e.g.: add_executable(tests_TriplePattern query/tests_TriplePattern.cpp)). Error occurring while adding executable in CMakeLists.txt